dependency/dependency_test.go

```diff
@@ -545,17 +563,25 @@ func (v *nomadServer) DeleteVariable(path string, opts *nomadapi.WriteOptions) e
 func (c *ClientSet) createConsulPartitions() error {
 	for p := range tenancyHelper.GetUniquePartitions() {
 		if p.Name != "" && p.Name != "default" {
-			partition := &api.Partition{Name: p.Name}
-			_, _, err := c.Consul().Partitions().Create(context.Background(), partition, nil)
-			if err != nil {
-				return err
+			err2 := c.createConsulPartition(p.Name)
```

Comment:
nit (can ignore): why err2?

Reply:
i think this was probably incidental from an IDE refactor or something.  will fix.

Reply:
resolved in edd0f7394427f64c6fc9de7aa74dd6bac9c32f80